### PR TITLE
Added rate request/response pojo and added rate API

### PR DESCRIPTION
**Pull request by manan-dft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# cario
```
Comment (dftrakesh) [inline README.md]: This is not cario. This is iconsignit

File: src/main/java/io/github/dft/kis/IconsIgnitSDK.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.SneakyThrows;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.concurrent.CompletableFuture;
+
+import static io.github.dft.kis.constantcode.ConstantCodes.*;
```
Comment (dftrakesh) [inline src/main/java/io/github/dft/kis/IconsIgnitSDK.java]: change package to io.github.dft.iconsignit

File: src/main/java/io/github/dft/kis/constantcode/ConstantCodes.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package io.github.dft.kis.constantcode;
+
+public interface ConstantCodes {
+    String CONTENT_TYPE = "Content-Type";
+    String ACCEPT = "Accept";
+    String HTTPS = "https://";
+    String BASE_ENDPOINT = "kis9.iconsignit.com.au/api/";
```
Comment (dftrakesh) [inline src/main/java/io/github/dft/kis/constantcode/ConstantCodes.java]: This should be %s.iconsignit.com.au/api/

And while constructing SDK. Take another parameter to take clientDomain

**Review by dftrakesh — CHANGES_REQUESTED** [review]
@manan-dft Please fix it.

File: README.md (hunk 1 above)
Comment (manan-dft) [reply README.md]: File updated with proper name

File: src/main/java/io/github/dft/kis/IconsIgnitSDK.java (hunk 2 above)
Comment (manan-dft) [reply src/main/java/io/github/dft/kis/IconsIgnitSDK.java]: Package name updated.

File: src/main/java/io/github/dft/kis/constantcode/ConstantCodes.java (hunk 3 above)
Comment (manan-dft) [reply src/main/java/io/github/dft/kis/constantcode/ConstantCodes.java]: Added parameter while building the SDK for consuming clientDomain.

**Pull request merged**